config/locales/en.yml

```diff
@@ -878,6 +873,9 @@ en:
     label: Course Activity
     last_update: Last statistics update
     next_update: Next estimated update
+    no_update: The Dashboard hasn't imported any data for this program yet
```

Comment:
This is a complete sentence and should end in a period.

Reply:
Dots are here. I can remove them to yml, but I would need to change other statistic messages too.
https://github.com/1v4n4/WikiEduDashboard/blob/0606c62b796d9c7450f416a8118b33c5f46cea76/app/assets/javascripts/components/overview/statistics_update_info.jsx#L88-L92

Reply:
Ah. Changing all of them would be good. Each complete message should include its own punctuation in the i18n file. If we rely on string substitution to combine messages with punctuation, it can result in garbled messages in languages where the English punctuation isn't relevant, or in right-to-left languages where the punctuation might end up on the wrong end of the message.